pkg/controllers/counter/controller.go

```diff
@@ -89,15 +86,67 @@ func (c *Controller) resourceCountsFor(provisionerName string) v1.ResourceList {
 		if n.MarkedForDeletion() {
 			return true
 		}
-		if n.Labels()[v1alpha5.ProvisionerNameLabelKey] == provisionerName {
+		if n.Labels()[ownerLabel] == ownerName {
 			res = resources.Merge(res, n.Capacity())
 		}
 		return true
 	})
 	return functional.FilterMap(res, func(_ v1.ResourceName, v resource.Quantity) bool { return !v.IsZero() })
 }
 
-func (c *Controller) Builder(_ context.Context, m manager.Manager) corecontroller.Builder {
+type NodePoolController struct {
+	*Controller
+}
+
+func NewNodePoolController(kubeClient client.Client, cluster *state.Cluster) corecontroller.Controller {
```

Comment:
Do we need to update the static drift annotation for nodepools?

Reply:
Yep, we need to update the drift annotation. We'll have that change in a separate PR